weatherApp.js

```diff
@@ -18,7 +18,8 @@ const search = ()=> {
 	.then(data => {
 		const dayBoxes = createDayBoxes(data);
 		renderDivlist(dayBoxes, "daylist");
-		const hourBoxes = createHourBoxes(Object.values(data)[0]);
+		log("data", data)
```

Comment:
Get rid of that log already. Use console.logs instead.
Or don't leave console.logs hanging in the code

Reply:
Ok, corrected to console.log